tsconfig.app.json

```diff
@@ -1,9 +1,11 @@
 {
-  "extends": "./tsconfig.base.json",
+  "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": [],
+    "strictNullChecks": true
   },
   "files": ["src/main.ts", "src/polyfills.ts"],
-  "include": ["src/**/*.d.ts"]
+  "include": ["src/**/*.ts"],
```

Comment:
@aaron-steinfeld The previous include was only considering `.d.ts`. So, my change is needed, isn't it?

Reply:
the files above served as roots, the includes were just for ambient defs